openassessment/xblock/static/js/src/studio/oa_edit_assessment.js

```diff
@@ -621,3 +621,104 @@ OpenAssessment.EditExampleBasedAssessmentView.prototype = {
     validationErrors: function() { return []; },
     clearValidationErrors: function() {}
 };
+
+/**
```

Comment:
Nit: this isn't a valid JSDoc doc string (I know that the whole file uses this Python-like convention, but still...). I've been fixing a file at a time as I run into such problems, but if you'd prefer just don't introduce any new occurrences.

http://usejsdoc.org/about-getting-started.html